src/Psalm/Type/Atomic.php

```diff
@@ -602,11 +590,10 @@ abstract public function toPhpString(
         ?string $namespace,
         array $aliased_classes,
         ?string $this_class,
-        int $php_major_version,
-        int $php_minor_version
+        int $analysis_php_version_id
     ): ?string;
 
-    abstract public function canBeFullyExpressedInPhp(int $php_major_version, int $php_minor_version): bool;
+    abstract public function canBeFullyExpressedInPhp(int $analysis_php_version_id): bool;
```

Comment:
This could possibly be changed to `(int $major_or_id, ?int $minor = null)` and the method could interpret first parameter as version id when second is null. The signature would be quite ugly though, but it'll avoid the BC break.